### ISSUE TITLE
Roadmap question

### ISSUE THREAD

**Issue by githubfanster** [balbarak/WeasyPrint-netcore#4]
Hello! 
Was just wondering if support for WEasyPrint would continue to catch up with later releases of WEasyPrint? Looks like WEasyPrint latest version , 51, is the minimum version i'll need because of its support for running generated content.

Also, is this library limited to what WEasyPrint can do in command line mode only?

Thank you for your work! Hope it continues :-)

**Comment by balbarak** [comment]
Hi, 
The support will continue and a new release will ship soon. Currently the library is limited to WEasyPrint command line. If you have any idea or suggestion feel free to contribute or open new issue

Thanks

**Comment by githubfanster** [comment]
Just a thought. If you're interested in deeper integration between your library and WEasyPrint, you might want to consider investigating using Python for .Net to call WEasyPrint api

https://github.com/pythonnet/pythonnet/issues/984

I myself have not used Python for .Net but I sure am hoping there would be C# bindings for WEasyPrint just as there are now C# bindings for  wkhtmltopdf and PuppeteerSharp for Puppeteer (i'm currently using PuppeteerSharp to generate PDFs)

**Comment by balbarak** [comment]
WeaspyPrint depends on GTK3 library which need extra work to handle not only python

I am going to look for it

Thank you for your suggestion.
